### ISSUE TITLE
Add/Modify Password in logins.json

### ISSUE THREAD

**Issue by islamadel** [lclevy/firepwd#23]
Ist there a script to modify or add new password to logins.json? Thanks

**Comment by lclevy** [comment]
No, but since des3 is a symmetric algorithm, it should be easy using existing functions
